v2/conn.go

```diff
@@ -768,63 +723,133 @@ func (c *Conn) RemoveSessionByIMSI(imsi string) {
 //
 // TODO: optimize performance...
 func (c *Conn) NewFTEID(ifType uint8, v4, v6 string) (fteidIE *ies.IE) {
-	c.mu.Lock()
-	defer c.mu.Unlock()
+	var teid uint32
+	// TODO: Name max retries
+	for i := 0; i < 100; i++ {
```

Comment:
100 times of retry seems insufficient in case of millions of subscribers exist. How about using 0xffff instead of 100 considering the unluckiest situation(only one value remaining unassigned)? Also, logging to let user know it's taking time at some point(e.g., when i = 1000, 10000, ...) can be helpful.